announcement/announcement-api/api/src/bundle/announcement.properties

```diff
@@ -25,6 +25,7 @@ java.alert.youacc.pes = You don't have permissions to access the message(s) - {0
 java.alert.thisis = This item is being edited by another user. Please try again later.
 ### (moot) java.alert.youcreate = you don't have permissions to create this announcement -
 java.alert.youchoosegroup = You need to choose at least one group.
+java.alert.youchooserole = You need to choose at least one role.
```

Comment:
Put all the new strings together at the end of the file, put a comment for S2U-36.